src/timer/timer_game_calendar.h

```diff
@@ -13,6 +13,78 @@
 #include "stdafx.h"
 #include "../core/strong_typedef_type.hpp"
 
+/**
+ * Storage class for Calendar time definition and constants.
+ */
+class CalendarTime {
+public:
+	/** The type to store our dates in. */
+	using Date = StrongType::Typedef<int32_t, struct DateTag, StrongType::Compare, StrongType::Integer>;
+
+	/** The fraction of a date we're in, i.e. the number of ticks since the last date changeover. */
+	using DateFract = uint16_t;
+
+	/** Type for the year, note: 0 based, i.e. starts at the year 0. */
+	using Year = StrongType::Typedef<int32_t, struct YearTag, StrongType::Compare, StrongType::Integer>;
+	/** Type for the month, note: 0 based, i.e. 0 = January, 11 = December. */
+	using Month = uint8_t;
+	/** Type for the day of the month, note: 1 based, first day of a month is 1. */
+	using Day = uint8_t;
```

Comment:
Why did you also move the types? That still means you can't use the functions. That was not what my suggestion was about :)

I would suggest: leave the types and functions in TimerGameCalendar, and ONLY have the constants in this class. And move it below TimerGameCalendar. This way you don't have `701266`, but you can just do `TimerGameCalendar::DateAtStartOfYear(ORIGINAL_BASE_YEAR)`, which was kinda the point of my idea :D